src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -187,13 +193,19 @@ object PlayLibraryBase extends AutoPlugin {
   import PlayBuildBase.autoImport._
 
   override def projectSettings = Seq(
+    // In sbt 1 the scripted plugin is automatically enabled. It imports
+    // its own jars into the project as a dependency. This fails for Scala
+    // 2.11 projects so we use this hack to remove the JARs again. :/
+    libraryDependencies := libraryDependencies.value.filter { m =>
```

Comment:
Anyway, using `filterNot` here could make the code slightly more clear.

Reply:
Removed this code since your suggestion above for disabling the plugin was superior.